packages/clay-charts/src/ChartBase.js

```diff
@@ -77,6 +76,7 @@ const ChartBase = {
 		}
 
 		this._resolveData(this.data).then(data => {
+			this._setupPolling();
```

Comment:
This should go to `DataComponent`, as `ChartBase` has no knowledge at all about polling

Reply:
Maybe this could go inside the `_resolveData` method, right before resolving the `Promise`?